docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -10,7 +10,8 @@ New and Improved
 
 - Peaks can now be added or removed from a PeaksWorkspace using the :ref:`peaks overlay <sliceviewer_peaks_overlay>` in :ref:`sliceviewer`.
 - The list of eligible workspaces in the `WorkspaceSelector` is now sorted alphabetically
-- New widget and workbench plugin: `WorkspaceCalculator`, allows to perform binary operations and scaling by a floating number on workspaces
+- New widget and workbench plugin: `WorkspaceCalculator`, allows to perform binary operations and scaling by a floating number on workspaces;
+  SAVE_STATE_VERSION has been incremented to 2
```

Comment:
It would be more useful for the end-user to have a description of an observable change, rather than an implementation detail. See, for example, the release notes here:
https://github.com/mantidproject/mantid/pull/31194/files

Reply:
That is absolutely a fair point. Then, I am going to repurpose most of the informative message and correct the last commit.